pyfolio/tests/test_perf_attrib.py

```diff
@@ -75,124 +75,6 @@ def mock_transactions_from_positions(positions):
 
 class PerfAttribTestCase(unittest.TestCase):
 
-    def test_perf_attrib_simple(self):
```

Comment:
Do we want to keep some portion of this as a regression test so that we don't break this by making an incompatible change in empyrical?  cc @twiecki @richafrank thoughts on this?

Reply:
I think my inclination is to keep these, given that we're still doing a somewhat nontrivial calculation in pyfolio (stacking the positions) before passing positions into empyrical.  Given that we already have these tests here, and that they're not very expensive, I don't see much cost to having them here.

Reply:
added back these tests

Reply:
I buy your point about testing this to ensure what pyfolio is doing to positions before passing them to empyrical works.